packages/core-data/src/actions.js

```diff
@@ -159,7 +159,7 @@ export function* editEntityRecord( kind, name, recordId, edits, options = {} ) {
 			const recordValue = record[ key ];
 			const editedRecordValue = editedRecord[ key ];
 			const value = mergedEdits[ key ] ?
-				merge( {}, editedRecordValue, edits[ key ] ) :
```

Comment:
Was the behavior until now to always perform a deep merge? If yes should we provide a note in case someone was relying on the previous deep merge behavior?

Reply:
I don't think anyone noticed, it's not very useful to rely on it and it breaks things. Think about what would happen if the value was an array for example.

We should still add a note just in case though. Where should we do that?